app/models/concerns/remotable.rb

```diff
@@ -10,7 +10,11 @@ module Remotable
       method_name = "#{attribute_name}=".to_sym
 
       define_method method_name do |url|
-        parsed_url = Addressable::URI.parse(url).normalize
+        begin
+          parsed_url = Addressable::URI.parse(url).normalize
```

Comment:
`Addressable::URI::InvalidURIError` is raised when executing `Addressable::URI.parse` with an illegal URL as an argument.